src/main/java/org/broadinstitute/consent/http/service/EmailService.java

```diff
@@ -180,17 +181,20 @@ public void sendNewDARCollectionMessage(Integer collectionId)
               + collectionId);
       return;
     }
-    List<User> distinctUsers = getDistinctAdminAndChairUsersForCollection(collection);
+    // Do this, but only for a single DAR
+    DataAccessRequest dar = collection.getMostRecentDar();
```

Comment:
This approach seems fine, but I think this should be a method in `DarCollectionService`, not in `EmailService`, since it has specific knowledge about the dar collection business logic.